ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -222,19 +222,6 @@
         "message": "The query used to build `OUTPUT` must include the join expression ROWKEY in its projection."
       }
     },
-    {
-      "name": "full-full missing join expression in projection",
```

Comment:
This test was invalid to begin with, because stream-table full outer joins are not supported.